Remotion/Web/Development.WebTesting/ControlSelection/IControlSelectionCommand.cs

```diff
@@ -32,8 +31,8 @@ public interface IControlSelectionCommand<out TControlObject>
     /// Performs the selection within the given <paramref name="context"/>.
     /// </summary>
     /// <returns>The <see cref="ControlObject"/> for the selected control.</returns>
-    /// <exception cref="AmbiguousException">If multiple matching controls are found but the specific implementation of <see cref="IControlSelectionCommand{TControlObject}"/> requires an exact match.</exception>
-    /// <exception cref="MissingHtmlException">If the element cannot be found.</exception>
+    /// <exception cref="WebTestException">If multiple matching controls are found but the specific implementation of <see cref="IControlSelectionCommand{TControlObject}"/> requires an exact match.</exception>

```

Comment:
We should probably merge this into a single Exception doc

Reply:
We found docs using this style.